ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/plugin/PluginImport.java

```diff
@@ -312,4 +312,16 @@ public void writeDelimeter(PrintWriter writer) {
 		writer.print(' ');
 	}
 
+	@Override
+	public void setFilter(String filter) throws CoreException {
+		// TODO Auto-generated method stub
+
+	}
+
+	@Override
+	public String getFilter() {
+		// TODO Auto-generated method stub
+		return null;
```

Comment:
This get and set method have to be implemented. But this time the value should be stored in a filed, as you currently do it in `PluginImportNode`.